.github/workflows/run-tests.yml

```diff
@@ -292,6 +292,7 @@ jobs:
           echo "TH_SNOW_URL=jdbc:snowflake://http://snowflake.localhost.localstack.cloud:4566" >> $GITHUB_ENV
 
       - name: Run Tests
+        if: github.ref == 'refs/heads/master' || matrix.testSystem != 'snowflake'
```

Comment:
Does this need the `${{`? I don't know enough about GHA to know if this is valid without it.

Reply:
yes, forgot in a hurry!!